baybe/acquisition/adapter.py

```diff
@@ -13,54 +11,7 @@
 from baybe.surrogates.base import Surrogate
 
 
-def debotorchize(acqf_cls: type[BotorchAcquisitionFunction]):
-    """Wrap a given BoTorch acquisition function.
-
-    This wrapped function becomes generally usable in combination with other non-BoTorch
-    surrogate models. This is required since BoTorch's acquisition functions expect a
-    ``botorch.model.Model`` to work with, hindering their general use with arbitrary
-    probabilistic models. The wrapper class returned by this function resolves this
-    issue by operating as an adapter that internally creates a helper BoTorch model,
-    which serves as a translation layer and is passed to the selected BoTorch
-    acquisition function, carrying the posterior information provided from any other
-    probabilistic model implementing BayBE's `Surrogate` interface.
-
-    Args:
-        acqf_cls: An arbitrary BoTorch acquisition function class.
-
-    Returns:
-        A wrapped version of the class that accepts non-BoTorch surrogate models.
-    """
-
-    class Wrapper:
-        """Adapter acquisition function that accepts BayBE surrogate models.
-
-        Args:
-            surrogate: The surrogate model that is being wrapped.
-            best_f: The best found objective function value found so far.
-        """
-
-        def __init__(self, surrogate: Surrogate, best_f: float):
-            self.model = AdapterModel(surrogate)
-            self.best_f = best_f
-
-            required_params = {
-                p: v
-                for p, v in {"model": self.model, "best_f": self.best_f}.items()
-                if p in signature(acqf_cls).parameters
-            }
-            self.botorch_acqf = acqf_cls(**required_params)
-
-        def __call__(self, candidates):
-            return self.botorch_acqf(candidates)
-
-        def __getattr__(self, item):
-            return getattr(self.botorch_acqf, item)
-
-    return Wrapper
-
-
-class AdapterModel(Model):
+class _AdapterModel(Model):
```

Comment:
Suggestion: instead of making the class private, let's make the entire module private (and with it all other acqf-related modules that are not user-facing, i.e. `partial.py`). I think we should slowly start doing this also in other places of the code, whenever we work on the corresponding parts